src/hal/layout_common.ld

```diff
@@ -2,6 +2,8 @@
 __aeabi_unwind_cpp_pr0 = abort;
 __aeabi_unwind_cpp_pr1 = abort;
 __aeabi_unwind_cpp_pr2 = abort;
+__exidx_start          = abort;
```

Comment:
Those should be defined in sections, not as symbols. I guess placing them after `_edata` is reasonable, to some extent.